frontend/packages/dev-console/src/components/gitops/GitOpsListPage.tsx

```diff
@@ -38,12 +41,39 @@ const GitOpsListPage: React.FC = () => {
     };
   }, [baseURL, namespaces, nsError, nsLoaded]);
 
+  const [consoleLinks] = useK8sWatchResource<K8sResourceKind[]>({
+    isList: true,
+    kind: referenceForModel(ConsoleLinkModel),
+    optional: true,
+  });
+  const argocdLink = _.find(
+    consoleLinks,
+    (link: K8sResourceKind) =>
+      link.metadata?.name === 'argocd' && link.spec?.location === 'ApplicationMenu',
+  );
+
   return (
     <>
       <Helmet>
         <title>Application Stages</title>
       </Helmet>
-      <PageHeading title="Application Stages" badge={<DevPreviewBadge />} />
+      <PageHeading
+        title="Application Stages"
+        badge={
+          <Split className="odc-gitops-list-page-heading" hasGutter>
+            <SplitItem>
+              {argocdLink && (
+                <ExternalLink
+                  href={argocdLink.spec.href}
+                  text="Argo CD"
+                  additionalClassName="odc-gitops-list-page-heading__argocd"
+                />
+              )}
+              <DevPreviewBadge />
+            </SplitItem>
```

Comment:
```suggestion
            <SplitItem>
              {argocdLink && (
                <ExternalLink
                  href={argocdLink.spec.href}
                  text="Argo CD"
                />
              )}
            </SplitItem>
            <SplitItem>
              <DevPreviewBadge />
            </SplitItem>
```

Reply:
The `hasGutter` property was not adding the required space between the two items since you wrapped both the items in a single `SplitItem` comp. As suggested above divide the items into two `SplitItem` comps and remove the css for adding space between the two.